classes/student.py

```diff
@@ -92,8 +101,17 @@ def filter_by_grade(all_students, threshold):
 
 
 if __name__ == "__main__":
-    csv_file = r'D:\Work\bac_stats\Data\good_bac_2019.csv'
-    all_students = initialiaze_students(csv_file)
-
+    
+    results_csv_file = r'/home/sebastian/Dropbox/Facultate/BacStats/BAC_2019_statistics/data/2019/good_bac_2019.csv'
+    schools_csv_file = r'/home/sebastian/Dropbox/Facultate/BacStats/BAC_2019_statistics/data/2019/unitati_scolare_2019.csv'
+
+    highschools = create_dictionary(schools_csv_file)
```

Comment:
this should be in initialize_students with an extra parameter because if you call initialize parameters now from another file, it won't work. Also the student constructor must accept also the old format because we may not have always access to the highschools data.